CONFIG.md

```diff
@@ -0,0 +1,232 @@
+  - Mixture Density Networks ([`mdn`](https://github.com/mackelab/sbi/blob/6c4fa7a6fd254d48d0c18640c832f2d80ab2257a/sbi/neural_nets/mdn.py#L14))
+  - Masked Autoregressive Flows ([`maf`](https://github.com/mackelab/sbi/blob/6c4fa7a6fd254d48d0c18640c832f2d80ab2257a/sbi/neural_nets/flow.py#L97))
+  - Neural Spline Flows ([`nsf`](https://github.com/mackelab/sbi/blob/6c4fa7a6fd254d48d0c18640c832f2d80ab2257a/sbi/neural_nets/flow.py#L293))
+  - Masked Autoencoders for Density Estimation ([`made`](https://github.com/mackelab/sbi/blob/6c4fa7a6fd254d48d0c18640c832f2d80ab2257a/sbi/neural_nets/flow.py#L24)). 
+- `SNRE` models use [`sbi.utils.classifier_nn`](https://github.com/mackelab/sbi/blob/6c4fa7a6fd254d48d0c18640c832f2d80ab2257a/sbi/utils/get_nn_models.py#L19) and support:
+  - Linear classifiers ([`linear`](https://github.com/mackelab/sbi/blob/6c4fa7a6fd254d48d0c18640c832f2d80ab2257a/sbi/neural_nets/classifier.py#L85))
+  - Multilayer Perceptrons ([`mlp`](https://github.com/mackelab/sbi/blob/6c4fa7a6fd254d48d0c18640c832f2d80ab2257a/sbi/neural_nets/classifier.py#L136))
+  - ResNets ([`resnet`](https://github.com/mackelab/sbi/blob/6c4fa7a6fd254d48d0c18640c832f2d80ab2257a/sbi/neural_nets/classifier.py#L194)).
+
+The **embedding_net** configuration allows one to specify additional neural layers which will prepend the input layer of the above neural density estimators. The default `sbi` architectures listed above are generally quite shallow, so its a good idea to make use of embedding architectures, especially for complex data. We include a fully-connected network (`ili.embedding.FCN`), but we also have an example of a CNN-like embedding network in [tutorial.ipynb](notebooks/tutorial.ipynb).
+
+The **train_args** are used to configure the training optimizer and early stopping criterion. All `sbi` models use the Adam optimizer. Lastly, **device** specifies whether to use Pytorch's `cpu` or `cuda` backend, and **output_path** specifies where to save your models after they are done training.
+
+Notes: Newly added `sbi` training engines, such as `MNLE`, don't yet work in our framework. Also, the `nsf` and `made` architectures unfortunately don't work if you're using `cuda` for GPU acceleration (yet!).
+
+
+### SBIRunnerSequential
+`SBIRunnerSequential` differs from `SBIRunner` in that it allows for multiround sampling for active learning inference. The configuration for `SBIRunnerSequential` is almost exactly the same as that of `SBIRunner`, except there is an added customizable parameter in `train_args` called `num_round` which specifies the number of rounds of simulation-training that the runner should do during training.
+
+### PydelfiRunner
+Here's an example configuration for `PydelfiRunner`
+```bash
+n_params: 3
+n_data: 10
+
+prior:
+  module: 'pydelfi.priors'
+  class: 'Uniform'
+  args:
+    lower: [0,0,0]
+    upper: [1,1,1]
+
+model:
+  module: 'ili.inference.pydelfi_wrappers'
+  class: 'DelfiWrapper'
+  kwargs:
+    nwalkers: 20 (FIX ME?)
```

Comment:
Why is there the entry (FIX ME?)?

Reply:
I wanted to check that this is the same as the tutorial example, and indeed it is. Removed (FIX ME?)